charts/opentelemetry-collector/README.md

```diff
@@ -107,6 +109,44 @@ agentCollector:
     mountPropagation: HostToContainer
 ```
 
+### Configuration for Kubernetes container logs
+
+The collector is preconfigured to receive logs sent to standard output by Kubernetes containers.
+
+This feature needs agent collector to be deployed, which means it will not work if only standalone collector is enabled.
+
+The feature can be disabled by setting the `agentCollector.containerLogs.enabled` property to `false`.
+
+The preconfigured container logs pipeline uses `logging` console exporter by default.
+To prevent looping the logs exported from collector back into the logs receiver,
+the default configuration of the receiver excludes logs from the collector containers.
+To change that, replace the exporter on the logs pipeline and remove the exclude on the logs receiver.
+
+Here's an example `values.yaml` file that replaces the default `logging` exporter on the `logs` pipeline
+with a `file` exporter that writes the container logs to `/var/log/container-logs/container-logs.json`.
```

Comment:
> I believe using file exporter AND collecting own logs will most likely result in amplification and exploding of logs.

This is not accurate. Explosion of logs will happen only when using `logging` exporter that sends the logs to console (why is it not named `console` exporter is a mystery to me btw. 🤔).

Anyhow, I will rephrase this paragraph to warn more loudly about the logs explosion. 👍

Reply:
if the file exporter would write to a path that it tails, only then, it will explode.

Reply:
>This is not accurate. Explosion of logs will happen only when using logging exporter that sends the logs to console

You are right.

>if the file exporter would write to a path that it tails, only then, it will explode.

Correct. This is more difficult to do accidentally but also a danger. So, it would be good to warn about this too.

Reply:
I have added a "warning" section to the docs in https://github.com/open-telemetry/opentelemetry-helm-charts/pull/36/commits/f005dde14bfdd2971aad69a3c0adbe3233a78a97, please take a look.

Reply:
> Let's not advertise the `file` exporter yet. It is unstable and is typically used for local troubleshooting only. Can we show the example using some other exporter instead?

Changed to `otlphttp` exporter, please let me know if this is any better :)

Reply:
LGTM.